### ISSUE TITLE
workflows/Ingest_and_Upload_Media;step=2 - wrong params passed to uploadToken->upload()

### ISSUE THREAD

**Issue by jessp01** [kaltura/developer-platform#62]
Consider the code https://developer.kaltura.com:8443/workflows/Ingest_and_Upload_Media;step=2 produces:

```php
$result = $client->uploadToken->upload($ks, $format, $uploadTokenId, $fileData, $resume, $finalChunk, $resumeAt);
```

```ruby
results = client.upload_token_service.upload(ks, format, upload_token_id, file_data, resume, final_chunk, resume_at)
```

The signature for this function is:
```php
function uploadAction($uploadTokenId, $fileData, $resume = false, $finalChunk = true, $resumeAt = -1)
```

So `ks` and `format` should not be there.

The above PHP and Ruby samples are just for illustration, same is true to all code generators.

**Comment by bobby-brennan** [comment]
Thanks, fixed in e2436aafcfd96b66494af4f2d0b205994040d42a

**Comment by jessp01** [comment]
Confirmed.
